yarn-project/end-to-end/src/e2e_p2p_network.test.ts

```diff
@@ -60,7 +60,12 @@ describe('e2e_p2p_network', () => {
 
     const initialValidators = [EthAddress.fromString(account.address)];
 
-    ({ teardown, config, logger, deployL1ContractsValues } = await setup(0, { initialValidators, ...options }));
+    ({ teardown, config, logger, deployL1ContractsValues } = await setup(0, {
+      initialValidators,
+      l1BlockTime: 12,
```

Comment:
magic numbers used across tests, probably best to define them in one place to prevent future headache